shared/dto/surveys/question-widget-map.entity.ts

```diff
@@ -1,10 +1,10 @@
-import { Entity, PrimaryColumn } from 'typeorm';
+import { Column, Entity, PrimaryColumn } from 'typeorm';
 
 @Entity('question_indicator_map')
 export class QuestionIndicatorMap {
   @PrimaryColumn()
   indicator: string;
 
-  @PrimaryColumn()
+  @Column()
```

Comment:
As mentioned in a previous comment, I totally forgot what is the point of having this table, given we reference both column
 values in the answers